src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+
+  if (days >= 7) {
+    return (sum -= 50);
```

Comment:
Use constants for the number of days and the discount amount to avoid hardcoding literals in expressions. This will make the code more readable and maintainable.